### PR TITLE
Add guide for migration from google analytics

### PR DESCRIPTION
**Pull request by kevinpagtakhan**
# Amplitude Developer Docs PR

## Description

Add guide for migration from google analytics

URL: https://pr-933.d19s7xzcva2mw3.amplifyapp.com/data/tutorials/migrating-from-google-analytics/

TODO: Update SDK snippet when package is published

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [x] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/tutorials/migrating-from-google-analytics/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+gtag('config', 'GA_MEASUREMENT_ID', {
+    user_id: 'USER_ID',
+});
+```
+
+Amplitude
+
+```js
+amplitude.setUserId('USER_ID');
+```
```
Comment (justin-fiedler) [inline docs/data/tutorials/migrating-from-google-analytics/index.md:76]: What about using diff views instead like the other migration docs?

**Review by justin-fiedler — APPROVED** [review]
Thanks @kevinpagtakhan =. Overall looks good. See comments.

File: docs/data/tutorials/migrating-from-google-analytics/index.md (hunk 1 above)
Comment (kevinpagtakhan) [reply docs/data/tutorials/migrating-from-google-analytics/index.md:76]: I tried that initially but found that it was a bit overwhelming when the entire block is being replaced. IMO the diff blocks are more useful when only a subset of the function call is changed.

File: docs/data/tutorials/migrating-from-google-analytics/index.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', 'G-XXXXXXXXXX');
+</script>
+```
+
+Then the corresponding Amplitude call is this:
+
+```html
+<script type="text/javascript">
```
[the remark this answers is not in the document]
Comment (kevinpagtakhan) [reply docs/data/tutorials/migrating-from-google-analytics/index.md:31]: Ohh i like that. I'll add that in a bit. We'll have the snippet soon.

File: docs/data/tutorials/migrating-from-google-analytics/index.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+Migrating from the Google Analytics (GA4) SDK to the Amplitude SDK involves adjusting your tracking implementation to align with Amplitude's event structure and capabilities.
+
+!!!info "Looking to get started easily with low-code migration?"
+
+    While completely replacing Google Analytics (GA4) tracking functions with Amplitude tracking functions is our preferred long-term solution for migrating, Amplitude recognizes that this process can be complex. Amplitude offers a [Google Analytics event forwarder](#using-google-analytics-events-forwarder) that can get you started with your migration quickly, with low-code interaction required from you.
+
+### Initializing Amplitude
+
+Replace the Google Analytics (GA4) initialization calls with Amplitude initialization calls in your application code. If you were tracking an event in Google Analytics (GA4) like this:
```
Comment (AlecBoutin) [inline docs/data/tutorials/migrating-from-google-analytics/index.md:16]:
```suggestion
Replace the Google Analytics (GA4) initialization calls with Amplitude initialization calls in your application code. If you are configuring Google Analytics (GA4) like this:
```

File: docs/data/tutorials/migrating-from-google-analytics/index.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+  amplitude.add('YOUR_API_KEY');
+</script>
+```
+
```
Comment (AlecBoutin) [inline docs/data/tutorials/migrating-from-google-analytics/index.md:37]: Should include link to all installation methods
```suggestion
For NPM and other installation methods click here.
```

File: docs/data/tutorials/migrating-from-google-analytics/index.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+Google Analytics (GA4)
+
+```js
+gtag('event', 'event_name', {
+  param1: 'value1',
+  param2: 'value2',
+});
+```
+
+Amplitude
```
Comment (AlecBoutin) [inline docs/data/tutorials/migrating-from-google-analytics/index.md:53]: Imo should include links to main SDK docs for all of these methods

**Review by AlecBoutin — APPROVED** [review]

File: docs/data/tutorials/migrating-from-google-analytics/index.md (hunk 5 above)
Comment (kevinpagtakhan) [reply docs/data/tutorials/migrating-from-google-analytics/index.md]: Updated!

File: docs/data/tutorials/migrating-from-google-analytics/index.md (hunk 4 above)
Comment (kevinpagtakhan) [reply docs/data/tutorials/migrating-from-google-analytics/index.md:37]: Updated!

File: docs/data/tutorials/migrating-from-google-analytics/index.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+#### Using Google Analytics events forwarder
+
+Amplitude offers a Google Analytics events forwarder which is a low-code solution that works as a plugin to the Amplitude Browser SDK. Once this plugin is installed on your application, it listens for events that Google Analytics (GA4) tracks. For each event sent to Google Analytics (GA4), a corresponding event is sent to Amplitude.
+
+##### Initializing Amplitude with Google Analytics events forwarder
+
+Add Amplitude SDK with Google Analytics events forwarder snippet right before your Google Tag snippet. Adding it before ensures that all Google Analytics (GA4) events are forwarded to Amplitude.
+
+```html
+<script type="text/javascript">
```
Comment (kevinpagtakhan) [inline docs/data/tutorials/migrating-from-google-analytics/index.md:122]: TODO: update version after public release

**Pull request merged**